packages/frontend/src/components/select_sports.tsx

```diff
@@ -0,0 +1,61 @@
+import { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+
+export function SelectSport() {
+  const [sport, setSport] = useState("Tennis");
+  const [colour, setColour] = useState("tennis");
```

Comment:
Use `Sport.tennis`, it would also be good to have this as undefined. (this means you will have to add an `option` like "Please select a sport` and then is unselectable later on.